DER/DerivativesContracts/Forwards.rdf

```diff
@@ -242,16 +390,42 @@
 		<rdfs:label xml:lang="en">open time</rdfs:label>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-der-der-fwd;UnderlyingValuation">
+	<owl:Class rdf:about="&fibo-der-der-fwd;StandardizedFuturesListingTerms">
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-fi-fi;StandardizedTerms"/>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-der-der-fwd;undertakenBy"/>
-				<owl:someValuesFrom rdf:resource="&fibo-der-der-fwd;CalculationAgent"/>
+				<owl:onProperty rdf:resource="&fibo-be-fct-pub;hasPublisher"/>
+				<owl:someValuesFrom rdf:resource="&fibo-der-drc-bsc;DesignatedContractMarket"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
-		<rdfs:label xml:lang="en">underlying valuation</rdfs:label>
-		<skos:definition xml:lang="en">The act of valuation of the underlying on some OTC Derivative.</skos:definition>
-		<skos:editorialNote xml:lang="en">Abstracted from FpML term for calculation Agent and Valuation terms as referred to in Equity Forward and presumably other derivatives also.</skos:editorialNote>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;appliesTo"/>
+				<owl:someValuesFrom rdf:resource="&fibo-der-der-fwd;FuturesListing"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">standardized futures listing terms</rdfs:label>
+		<skos:definition xml:lang="en">Standard terms out by the exchange which will apply to any listing of a futures contract on this exchange.</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">Individual listings will take on these standard terms but they are not contractual terms of the futures contract, they are facts about that listing on that exchange.</fibo-fnd-utl-av:explanatoryNote>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-der-der-fwd;StandardizedFuturesTerms">
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-fi-fi;StandardizedTerms"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fbc-fct-ra;specifies"/>
+				<owl:someValuesFrom rdf:resource="&fibo-der-drc-bsc;DeliveryTerms"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;appliesTo"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fbc-fi-fi;Future"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">standardized futures terms</rdfs:label>
+		<skos:definition xml:lang="en">A set of standard terms for futures contracts, as set out by the derivatives exchange. Futures contracts issued by and traded on that exchange will take on these terms or override them.</skos:definition>
```

Comment:
where is the link to the Exchange? Though it should really be a link FROM the Exchange.